apis/Google.Cloud.Spanner.Data/docs/configuration.md

```diff
@@ -142,11 +142,23 @@ server, including retries. This setting is applied to calls to
 create, refresh and delete sessions, as well as beginning
 transactions.
 
-Applications rarely need to change this setting
+Applications rarely need to change this setting.
+
+**CreateSessionBatchMaximumSize**
```

Comment:
Maybe this should be CreateSessionMaximumBatchSize? That sounds better to me (in that it's the "maximum batch size" for "create session" but I could probably be convinced :)

Reply:
Done